responsibilities/community_meeting.md

```diff
@@ -20,48 +22,58 @@ At KubeVirt's present scale the community opts to host a weekly meetings for all
   * Community organizers will attempt to transcribe each topic in the agenda
 
 * Recordings
-  * Community organizers will process Zoom mp4 digital audio/video recording into a recording using Open Source codecs [VP9](https://en.wikipedia.org/wiki/VP9) video and [Opus](https://en.wikipedia.org/wiki/Opus_%28audio_format%29) audio
-```
-  cd zoom/DATE
-  ffmpeg -i "${I}" -c:v libvpx-vp9 -c:a libopus $(basename "`pwd`" | awk {'print $1 "_" $3 "_" $4 "_" $5'}).webm
-```
-  * Community organizers will upload the video to KubeVirt’s YouTube channel and the Community Meetings playlist.
+  * The Zoom meeting is set to record to the platform
+  * Recording needs to be dowloaded then uploaded to KubeVirt’s YouTube channel
```

Comment:
```suggestion
  * Recording needs to be downloaded then uploaded to KubeVirt’s YouTube channel
```

Reply:
It's almost like we need spell check on the community repo.... ;)

Fixed